Description: Add .TB macro to manpage. This macro is extracted from tmac/man.ultrix present in groff upstream source. See http://www.gnu.org/software/groff/manual/html_node/Optional-man-extensions.html#Optional-man-extensions
Author: Mònica Ramírez Arceda <monica@probeta.net>
Last-Update: 2011-02-13

--- a/splitvt.1
+++ b/splitvt.1
@@ -1,3 +1,15 @@
+.if n .ig
+.de HB
+.ie \\n[.$] .ft HB
+.el \f[HB]\\$*\fP
+..
+.if t .ig
+.de HB
+.ie \\n[.$] .ul 999
+.el .ul
+\\$*
+..
+.als TB HB
 .TH SPLITVT 1 "1/5/95" "SPLITVT" "NONESSENTIAL UTILITIES"
 
 .SH NAME
